h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
   max-width: 500px;
+  flex-shrink: 0;
   padding-left: 0;
   padding-right: 0;
 }
 
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
```

Comment:
This is what was causing the unnecessary wrapping of stats, let flexbox expand the element to fill the width available instead